ProgramParsing/Math/ParseProgram.py

```diff
@@ -14,16 +14,22 @@
 
     filesToIgnore = ["ENG-Software-Engineering.html",
                      "MATH-Mathematical-Optimization1.html", "MATH-Math-or-Chartered-Professional-Accountancy-co.html"]
-    filesToIgnore = set(["/Specs/" + f for f in filesToIgnore])
+    # adds the academic year in front of strings in filesToIgnore
+    filesToIgnoreYear = []
+    # adds year in front of files in filesToIgnore
+    filesToIgnoreYear = add_year_ignore(filesToIgnore)
+
+    filesToIgnore = set(["/Specs/" + f for f in filesToIgnoreYear])
+    print(filesToIgnore)
     files = files - filesToIgnore
```

Comment:
- remove the print(filesToIgnore) statement
- I want line 17-24 be part of the add_year_ignore folder s.t you pass in (files, filesToIgnore) and get files (which is filtered): would prob rename add_year_ignore -> filterFiles or something like that
- Also replace all parsers (every faculty) with this new function you created (code will be so clean after this)